bin/broker_submission.py

```diff
@@ -15,13 +15,10 @@
 # limitations under the License.
 
 import logging
-import os
-import sys
 from argparse import ArgumentParser
 
 from ebi_eva_common_pyutils.logger import logging_config as log_cfg
 
-sys.path.append(os.path.dirname(os.path.dirname(__file__)))
```

Comment:
good riddance!

Reply:
now that i've actually looked at the deploy script - can't we also achieve this by adding `python setup.py install` rather than switching to `deploy_python_in_venv.sh`?  maybe there's other good reasons to switch but i'm just curious

Reply:
This is (roughly) what [this line](https://github.com/EBIvariation/eva-tools/pull/111/files#diff-a8c6f877230e2e450dc2c45f265806f4a4d31a6b620add1c3e317b216e11f8b0R59) is doing. 
Although `pip` is better at managing dependencies and will get the package directly from Github
Plus the rest of the script provide support for getting a branch or a tag and storing each in independent folders.